src/Uno.UWP/Devices/Midi/Internal/WasmMidiAccess.wasm.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using static Uno.Foundation.WebAssemblyRuntime;
+
+namespace Uno.Devices.Midi.Internal
+{
+	/// <summary>
+	/// Handles WASM MIDI access permission request
+	/// </summary>
+	internal static class WasmMidiAccess
+	{
+		private const string JsType = "Uno.Devices.Midi.Internal.WasmMidiAccess";
+
+		private static bool _webMidiAccessible = false;
+
+		internal static async Task<bool> RequestAsync()
+		{
+			if (_webMidiAccessible) return true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)